packages/host/app/components/operator-mode/code-submode/module-inspector.gts

```diff
@@ -414,6 +418,17 @@ export default class ModuleInspector extends Component<ModuleInspectorSignature>
     );
   }
 
+  private iconForView(view: ModuleInspectorView) {
+    switch (view) {
+      case 'schema':
+        return Schema;
+      case 'preview':
+        return Eye;
+      case 'spec':
+        return FileCog;
+    }
```

Comment:
maybe these should be set where the `moduleInspectorPanels` values are set

Reply:
done in 7b6cada, thanks